packages/admin/src/main.ts

```diff
@@ -1350,7 +1378,8 @@ class Admin extends utils.Adapter {
                                             }
                                         }
                                     }
-                                } catch (e) {
+                                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                                } catch (e: unknown) {
```

Comment:
you can just remove the unused ``(e)`` and just use `catch {`

Reply:
Done